frontend/src/components/navigation/FourZeroFourDisplay.vue

```diff
@@ -0,0 +1,200 @@
+      </linearGradient>
+      <radialGradient
+        id="radial-gradient-3"
+        cx="117.33"
+        cy="284.42"
+        fx="117.33"
+        fy="284.42"
+        r="243.01"
+        gradientUnits="userSpaceOnUse"
+      >
+        <stop offset=".8" stop-color="#e8decd" />
+        <stop offset="1" stop-color="#a8a39c" />
+      </radialGradient>
+      <radialGradient
+        id="radial-gradient-4"
+        cx="366.75"
+        cy="244.4"
+        fx="366.75"
+        fy="244.4"
+        r="133.71"
+        gradientUnits="userSpaceOnUse"
+      >
+        <stop offset="0" stop-color="#ffb48a" />
+        <stop offset=".6" stop-color="#ff9768" />
+        <stop offset="1" stop-color="#ff7e4a" />
+      </radialGradient>
+      <radialGradient
+        id="radial-gradient-5"
+        cx="1166.64"
+        cy="76.54"
+        fx="1166.64"
+        fy="76.54"
+        r="525.06"
+        xlink:href="#radial-gradient"
+      />
+      <linearGradient
+        id="linear-gradient-2"
+        x1="1116.44"
+        x2="1119.25"
+        y2="414.67"
+        xlink:href="#linear-gradient"
+      />
+      <radialGradient
+        id="radial-gradient-6"
+        cx="1021.3"
+        fx="1021.3"
+        r="243.01"
+        xlink:href="#radial-gradient-3"
+      />
+      <radialGradient
+        id="radial-gradient-7"
+        cx="1270.73"
+        fx="1270.73"
+        xlink:href="#radial-gradient-4"
+      />
+    </defs>
+    <g id="Layer_4" data-name="Layer 4">
+      <path
+        class="cls-7"
+        d="m487.37,378.96c0-185.18,77.73-279.93,194.67-279.93s194.67,95.46,194.67,279.93-77.73,284.77-194.67,284.77-194.67-99.59-194.67-284.77Zm270.14,0c0-148.47-33.05-182.52-75.47-182.52s-75.47,34.04-75.47,182.52,33.05,187.36,75.47,187.36,75.47-38.88,75.47-187.36Z"
+      />
+    </g>
+    <g id="FOUR">
+      <path
+        class="cls-3"
+        d="m374.06,108.51h-59.78l-225.64,415.57h347.54c-30.2-85.62-49.27-239.02-62.12-415.57Z"
+      />
+      <path
+        class="cls-8"
+        d="m85.95,524.09L311.59,108.51h55.98l3.82,319.18v96.4s-.01,87.11-.01,130.67h-119.96l-.1-130.62-165.36-.05Z"
+      />
+      <polygon
+        class="cls-10"
+        points="265.15 199 289.89 427.24 140.98 427.69 265.15 199"
+      />
+      <path
+        class="cls-9"
+        d="m265.67,203.77c15.94,90,31.14,167.54,58.77,212.26-8.22,6.56-19.16,11.33-35.27,11.22-20.28-47.37-26.76-129.74-23.5-223.48Z"
+      />
+      <path
+        class="cls-5"
+        d="m88.64,524.09H20.61v-91.15L219.83,108.51h154.23c-53.21,15.13-119.96,110.84-119.96,110.84l-165.46,304.73Z"
+      />
+    </g>
+    <g id="Layer_5" data-name="Layer 5">
+      <path
+        class="cls-3"
+        d="m1278.03,108.51h-59.78l-225.64,415.57h347.54c-30.2-85.62-49.27-239.02-62.12-415.57Z"
+      />
+      <path
+        class="cls-4"
+        d="m989.92,524.09l225.64-415.57h55.98l3.82,319.18v96.4s-.01,87.11-.01,130.67h-119.96l-.1-130.62-165.36-.05Z"
+      />
+      <polygon
+        class="cls-1"
```

Comment:
```suggestion
        fill="url(#linear-gradient-2)"
```
Please use attributes instead of css. It makes it more readable.